perception_launch/launch/perception.launch.xml

```diff
@@ -4,6 +4,7 @@
   <arg name="vehicle_param_file" />
   <!-- common parameters -->
   <arg name="mode" default="camera_lidar_fusion" description="options: `camera_lidar_fusion`, `lidar` or `camera`"/>
+  <arg name="input/pointcloud" default="/sensing/lidar/top/outlier_filtered/pointcloud"/>
```

Comment:
The default might as well be `/sensing/lidar//pointcloud`.
Some vehicles don't have a top lidar, and I think users should change it as needed.

Reply:
It seems good. I'll change it.

Reply:
@yukkysaito 
Users would like to set input topic of perception and localization module from root (autoware.launch ) so I think that removing all default value except autoware.launch and set `/sensing/lidar/lidar` as default value in autoware.launch is better. How do you think?

Reply:
I agree with you :+1:

Reply:
@yukkysaito I fixed it in https://github.com/tier4/autoware_launch/pull/215/commits/380660c0ddbade8e95c982dcf5e0fe37ee350854

Reply:
I don't like this because it doesn't launch `ros2 launch localization_launch localization.launch.xml`.

Reply:
@YamatoAndo Launching file with argument `ros2 launch localization_launch localization.launch.xml input_sensor_points_topic:=/sensing/lidar/pointcloud` is still available, is that unacceptable?

Reply:
yes. 
Why should we bother to specify `/sensing/lidar/pointcloud` as an argument when we know that it is the default?

Reply:
I think pros of this proposal is that input topic setting location is uniquely detemined. As a result, users can easily understand where it should be set.

Reply:
Do you want to change input topic of localization often?
And there are many other parameters in Autoware, do you need to change them in autoware.launch?

Reply:
> Do you want to change input topic of localization often?
We have to change input topic name for each product to adjust to actual sensor configuration.

Reply:
I can understand if all other parameters are also in autoware.launch, but I think it is hard to understand that only the change of input topic for localization is in autoware.launch.

Reply:
The input topic name of `perception.launch` is also detemined in autoware.launch and these topic will be declared by product sensor specification.

Are there any other module that need to detemine input topic name by product specification?

Reply:
Too many other input topics in perception.launch.
Why are these not written in autoware.launch?
https://github.com/tier4/autoware_launch/blob/9198bfb52d71bef4014da3ea1d3be5e51916db89/perception_launch/launch/perception.launch.xml#L7-L22

Reply:
The example is certainly what you say. However, I don't think the current launch file is good for this example, so I think we should improve the topic setting method so that it is easier to understand.

Anyway, I'm not very particular about it, so I'll fix launch file based on you suggestion.

Reply:
@YamatoAndo please refer to https://github.com/tier4/autoware_launch/pull/215/commits/e61de66229e500c0e0378c44296e402455b4bb01

Reply:
In my opinion, it should be one of the following.
1. Set the default value in localization.launch and perception.launch as before.
2. Set all input topics (and output topics if necessary)  of each module in autoware.launch.

Changing from autoware.launch is confusing, at least for me, so I think option 1 is better.

Reply:
Of course, if the majority of people agree with Higashide-san's idea, I'll follow it.

Reply:
> please refer to https://github.com/tier4/autoware_launch/commit/e61de66229e500c0e0378c44296e402455b4bb01

OK!!

Reply:
However this will eventually change it in autoware.launch.
It's a little confusing...

Reply:
@YamatoAndo 
> However this will eventually change it in autoware.launch.
It's a little confusing...

This is a point of my opinion. I think removing default argument from autoware and logging.launch is also confusing.

Reply:
I think it's better not to write topic names in autoware and logging.launch.

Reply:
@yukkysaito @kenji-miyake 
Could you decide to which is better?
1. Setting input topic name of localization and perception modue on autoware.launch
2. Setting it on each localization.launch and perception.launch

Reply:
(Although I haven't read this thread well,) I feel the No. 2 is better because Autoware components should be easily replaceable.

Reply:
I fixed it in https://github.com/tier4/autoware_launch/pull/215/commits/44e0c395bb7dc746b95bdb536245ffae5a610a6f, please resolve conversation

Reply:
LGTM